package.json

```diff
@@ -32,7 +32,7 @@
   ],
   "dependencies": {
     "@segment/loosely-validate-event": "^2.0.0",
-    "auto-changelog": "^1.16.2",
+    "auto-changelog": "2.3.0",
```

Comment:
Any specific reason for not using semver notation?

Reply:
You mean "^"?

Reply:
Done